apps/code-examples/src/app/code-examples/forms/input-box/basic/input-box-demo.component.html

```diff
@@ -8,156 +8,68 @@ <h2>New member form</h2>
-                  email.errors &&
-                  email.errors['skyEmail']
-                "
-                indicatorType="danger"
-                descriptionType="error"
-              >
-                Enter a valid email in the "joe@abc.com" format.
-              </sky-status-indicator>
-            </span>
           </sky-input-box>
         </sky-column>
         <sky-column [screenSmall]="12" [screenMedium]="6">
-          <sky-input-box class="sky-margin-stacked-lg">
-            <label class="sky-control-label" [for]="dobInput.id">
-              Date of birth
-            </label>
+          <sky-input-box labelText="Date of birth" stacked="true">
             <sky-datepicker>
-              <input
-                formControlName="dob"
-                [attr.aria-describedby]="dobError.id"
-                skyId
-                #dobInput="skyId"
-                skyDatepickerInput
-              />
+              <input formControlName="dob" skyDatepickerInput />
             </sky-datepicker>
-            <span class="sky-error-indicator" skyId #dobError="skyId">
-              <sky-status-indicator
-                *ngIf="
-                  (dob.dirty || dob.touched) &&
-                  dob.errors &&
-                  dob.errors['required']
-                "
-                indicatorType="danger"
-                descriptionType="error"
-              >
-                Date of birth is required.
-              </sky-status-indicator>
-            </span>
           </sky-input-box>
         </sky-column>
       </sky-row>
       <sky-row>
         <sky-column [screenSmall]="12">
-          <sky-input-box class="sky-margin-stacked-lg">
-            <label class="sky-control-label" [for]="favoriteColorInput.id">
-              Favorite color
-            </label>
-            <select
-              formControlName="favoriteColor"
-              skyId
-              class="sky-form-control"
-              #favoriteColorInput
-            >
+          <sky-input-box labelText="Favorite color">
+            <select formControlName="favoriteColor">
               <option value="none">None</option>
               <option value="red">Red</option>
               <option value="orange">Orange</option>
               <option value="yellow">Yellow</option>
               <option value="green">Green</option>
               <option value="blue">Blue</option>
               <option value="purple">Purple</option>
+              <option value="bird">Bird</option>
             </select>
+            <!-- Custom validator not handled by input box. -->
+            <div class="sky-error-indicator">
+              <sky-status-indicator
+                *ngIf="favoriteColor.errors?.['color'] as colorError"
+                descriptionType="error"
+                indicatorType="danger"
+              >
+                {{ colorError.message }}
+              </sky-status-indicator>
+            </div>
```

Comment:
I added a character count to the bio field in the basic example. I left the help inline example as-is because consumers may want to do something other than show a popover, like open a new window, and I wanted to demonstrate that that's still possible.